_includes/manuals/1.13/4.1.2_roles_and_permissions.md

```diff
@@ -33,6 +35,8 @@ Some example queries for the resource Host:
 
 These pools of queries can be combined by adding them together or the filters can be used to restrict the selected resource to a smaller and smaller subset of the total. Think of them as set operations.
 
+As already mentioned, Role can be assigned to Organizations and Locations. In such case, all filters for resources that support such scoping automatically apply the same Organizations and Locations. If you want to combine filters with different Organizations or Locations assignments, you can use 'Override' check box. When checked you can override Organizations and Location for a filter. If you uncheck this field, the filter starts inheriting its role Organizations and Locations after submitting again. If you want to reset all role's filter to start inheriting, you can use 'Disable all filters overriding' button on role's 'Filters' tab. We recommend managing Organizations and Locations association on Role level to keep the setup simple and clear.
```

Comment:
"*a* Role can be assigned to"
"if you want to reset all *role filters*"

A screenshot to show where 'disable all filters overriding' is located would help IMO, it's hard to tell from just text.